pwndbg/gdblib/arch.py

```diff
@@ -96,7 +96,12 @@ def _get_arch(ptrsize: int):
     for match in ARCHS:
         if match in arch:
```

Comment:
Given more and more if condition logic here, I wonder if we should maybe unroll the loop manually into sth like:

```py
if arch in SIMPLE_ARCHS:
    return arch, ptrsize, endian
# <comment>
elif "arm" in arch and ("-m" in arch or pwndbg.gdblib.regs.xpsr is not None):
    return "armcm", ptrsize, endian
...
```